datasets/new_york/pipelines/_images/run_csv_transform_kub_citibike_stations/csv_transform.py

```diff
@@ -140,125 +372,120 @@ def datetime_from_int(dt_int: int) -> str:
     return datetime.datetime.fromtimestamp(dt_int).strftime("%Y-%m-%d %H:%M:%S")
 
 
-def clean_data_points(df: pd.DataFrame) -> pd.DataFrame:
-    df = resolve_datatypes(df)
-    df = normalize_data_list(df)
-    df = resolve_boolean_datapoints(df)
-
+def clean_data_points(
+    df: pd.DataFrame,
+    resolve_datatypes_list: dict,
+    normalize_data_list: typing.List[str],
+    boolean_datapoints_list: typing.List[str],
+) -> pd.DataFrame:
+    df = resolve_datatypes(df, resolve_datatypes_list)
+    df = normalize_data(df, normalize_data_list)
+    df = resolve_boolean_datapoints(df, boolean_datapoints_list)
     return df
 
 
-def resolve_datatypes(df: pd.DataFrame) -> pd.DataFrame:
-    logging.info("Resolving datatypes")
-    columns = {"station_id": "Int64", "region_id": "Int64", "rental_methods": "string"}
-
-    for column, datatype in columns.items():
+def resolve_datatypes(df: pd.DataFrame, resolve_datatypes_list: dict) -> pd.DataFrame:
+    for column, datatype in resolve_datatypes_list.items():
+        logging.info(f"Resolving datatype for column {column} to {datatype}")
         df[column] = df[column].astype(datatype)
-
     return df
 
 
-def normalize_data_list(df: pd.DataFrame) -> pd.DataFrame:
-    logging.info("Normalizing data lists")
-    columns = ["rental_methods"]
-
-    for column in columns:
+def normalize_data(
+    df: pd.DataFrame, normalize_data_list: typing.List[str]
+) -> pd.DataFrame:
+    for column in normalize_data_list:
+        logging.info(f"Normalizing data in column {column}")
         df[column] = (
             str(pd.Series(df[column])[0])
             .replace("[", "")
             .replace("'", "")
             .replace("]", "")
```

Comment:
Please add a comment or an example of what's being replaced with empty strings

Reply:
Done